scripts/provision_cluster.py

```
@@ -0,0 +1,590 @@
+    raw: RawProvisionInputs,
+    _resolved: Callable[[str | Path | None, InputResolution], str | Path],
+    to_path: Callable[[Path | str], Path],
+) -> tuple[Path, Path, bool]:
+    """Resolve execution configuration inputs."""
+    # RUNNER_TEMP/GITHUB_ENV/DRY_RUN InputResolution defaults (Path("/tmp"),
+    # Path("/tmp/github-env-undefined"), and "false") are intentional
+    # local-dev/test fallbacks to avoid hard failures when those env keys are
+    # absent; production usage should set the env_key values explicitly.
+    runner_temp_raw = _resolved(
+        raw.runner_temp,
+        InputResolution(env_key="RUNNER_TEMP", default=Path("/tmp"), as_path=True),
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)